### PR TITLE
chore: update changelog for 1.8.6

### PR DESCRIPTION
**Pull request by psteinbachs**
Closes #

Updates changelog for 1.8.6

- [ ] Rebased/mergeable

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -52,13 +52,17 @@ v1.9.0 [unreleased]
 -	[#21358](https://github.com/influxdata/influxdb/pull/21358): fix: disable MergeFiltersRule until it is more stable
 -	[#21516](https://github.com/influxdata/influxdb/pull/21516): fix: FGA enablement
 
-v1.8.6 [unreleased]
+v1.8.6 [2021-05-21]
 -------------------
 
+Note: No OSS specific changes were made in this release.
+
 -	[#21290](https://github.com/influxdata/influxdb/pull/21290): fix: Anti-Entropy loops endlessly with empty shard
```
Comment (timhallinflux) [inline CHANGELOG.md:60]: if there are no OSS specific fixes...why are these three PRs listed?

File: CHANGELOG.md [hunk 2]
```diff
@@ -52,13 +52,17 @@ v1.9.0 [unreleased]
 -	[#21358](https://github.com/influxdata/influxdb/pull/21358): fix: disable MergeFiltersRule until it is more stable
 -	[#21516](https://github.com/influxdata/influxdb/pull/21516): fix: FGA enablement
 
-v1.8.6 [unreleased]
+v1.8.6 [2021-05-21]
 -------------------
 
+Note: No OSS specific changes were made in this release.
```
Comment (timhallinflux) [inline CHANGELOG.md:58]: add -> This release was created to support InfluxDB Enterprise 1.8.6.

**Review by timhallinflux — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (psteinbachs) [reply CHANGELOG.md:60]: The commits needed to be made to the OSS repo as a result of existing code organization.

File: CHANGELOG.md (hunk 2 above)
Comment (psteinbachs) [reply CHANGELOG.md]: Added.

**Review by timhallinflux — APPROVED** [review]

**Pull request merged**